tests/Resources/ti.media.test.js

```diff
@@ -4,6 +4,7 @@
  * Licensed under the terms of the Apache Public License
  * Please see the LICENSE included with this distribution for details.
  */
+/* global OS_IOS */
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.media.test.js;
  Line: 7;
-->

- :warning: tests/Resources/ti.media.test.js line 7 – 'OS_IOS' is defined but never used. (no-unused-vars)